src/controllers/playerController.ts

```diff
@@ -43,6 +42,125 @@ class PlayerController {
             await next();
         }
     }
-};
+
+    /**
+     * Gets a player from the database by ID
+     * @param {RouterContext} ctx The request context object containing data for the new player.
+     * @param {() => Promise<void>} next The next client request.
+     */
+    public async getPlayer(ctx: RouterContext, next: () => Promise<void>): Promise<void>{
+
+        // try/catch block allows us to capture errors to return to the client
+        try {
+            // Get player object and respond to client
+            ctx.body = await Player.findById(ctx.request.body.id);
```

Comment:
I know I said to get this from the ctx.request.body.id, but in get requests, we will use a url param. I will give you more information on this in discord.